{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf830
{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fnil\fcharset0 Menlo-Regular;}
{\colortbl;\red255\green255\blue255;\red200\green0\blue57;\red57\green237\blue255;}
{\*\expandedcolortbl;;\cssrgb\c83351\c5901\c28783;\cssrgb\c24328\c93682\c100000;}
\margl1440\margr1440\vieww10800\viewh8400\viewkind0
\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0

\f0\fs24 \cf0 \
\
Error trying to connect MySQL \
\
\
\pard\tx560\tx1120\tx1680\tx2240\tx2800\tx3360\tx3920\tx4480\tx5040\tx5600\tx6160\tx6720\pardirnatural\partightenfactor0

\f1\fs22 \cf2 \cb3 \CocoaLigature0   File "/Users/juancarloscarlson/Desktop/my_new_project/venv/lib/python2.7/site-packages/MySQLdb/connections.py", line 193, in __init__\
    super(Connection, self).__init__(*args, **kwargs2)\
sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (61)")\
\
mysql -u root -p\
Enter password: \
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2)\
(venv) JuanCarlosCarlsons-MacBook-Pro:flask_mysql juancarloscarlson$ \
\
==============================================\
\
\
juancarloscarlson$ sudo mysql_safe\
Password:\
sudo: mysql_safe: command not found\
(venv) JuanCarlosCarlsons-MacBook-Pro:flask_mysql juancarloscarlson$ \
\
}